k4ProjectTemplate/CMakeLists.txt

```diff
@@ -1,11 +1,13 @@
 
-
-
-
 file(GLOB _plugin_sources src/components/*.cpp)
 gaudi_add_module(k4ProjectTemplatePlugins
                  SOURCES ${_plugin_sources}
-                 LINK Gaudi::GaudiKernel Gaudi::GaudiAlgLib k4FWCore::k4FWCore k4FWCore::k4FWCorePlugins  EDM4HEP::edm4hep EDM4HEP::edm4hepDict)
+                 LINK Gaudi::GaudiKernel 
+                      Gaudi::GaudiAlgLib
+                      k4FWCore::k4FWCore
+                      EDM4HEP::edm4hep
+                      EDM4HEP::edm4hepDict
```

Comment:
One also doesn't have to link against the dict, right?

Reply:
```suggestion
```